src/Microsoft.AspNet.Mvc.DataAnnotations/DataAnnotationsClientModelValidatorProvider.cs

```diff
@@ -36,15 +30,12 @@ public class DataAnnotationsClientModelValidatorProvider : IClientModelValidator
         /// <param name="stringLocalizerFactory">The <see cref="IStringLocalizerFactory"/>.</param>
         public DataAnnotationsClientModelValidatorProvider(
             IOptions<MvcDataAnnotationsLocalizationOptions> options,
-            IStringLocalizerFactory stringLocalizerFactory)
+            IStringLocalizerFactory stringLocalizerFactory,
+            IValidationAttributeAdapterProvider validationAttributeAdapterProvider)
```

Comment:
Order is odd here too. Please move the `IValidationAttributeAdapterProvider` up and clean up the tests